fix: Make sure that document is available before reading location

kamilogorek opened the pull request:
Fixes https://github.com/getsentry/raven-js/issues/1037

https://github.com/getsentry/raven-js/pull/1038/files?w=1#diff-05367d6a2553cabeca57c6ca111093fcR34 – this is the only real change, the rest is just prettify catching up with the rest of the files through precommit hook

benvinegar commented on vendor/TraceKit/tracekit.js:
```diff
@@ -31,13 +31,11 @@ var UNKNOWN_FUNCTION = '?';
 function getLocationHref() {
-    if (typeof document === 'undefined' || typeof document.location === 'undefined')
-        return '';
+  if (document == null || document.location == null) return '';
```
This seems problematic. What if `document` isn't defined? Won't accessing it to make a null comparison throw an exception?

Why not do:

```javascript
if (typeof document === 'undefined' || document.location == null) return '';
```

kamilogorek replied:
Thanks for catching that. I completely forgot that `document` is just a reference to `window.document` and it has to obey the same rules as every other property.

kamilogorek commented:
Updated

benvinegar commented:
Approved, but in the future, we should try to separate autoformatting changes from behavior changes. It's worth having separate PRs.

kamilogorek commented:
I totally missed that @benvinegar, thanks for pointing that out. Fixed, merged code change only and updated PR here https://github.com/getsentry/raven-js/pull/1048
